README.rst

```diff
@@ -114,8 +114,8 @@ Android:
 
    <dependency>
      <groupId>org.web3j</groupId>
-     <artifactId>core</artifactId>
-     <version>3.0.1-android</version>
+     <artifactId>core-android</artifactId>
```

Comment:
From release 3.x onwards the Android releases are versioned using <module>:3.x.x-android. This is so that the modules all don't need to be renamed in the Android branch to <module>-android.

Reply:
Updated!